src/form/Input/Input.tsx

```diff
@@ -38,14 +39,28 @@ export interface InputProps
 
   /**
    * Content to display before the input.
+   *
+   * @deprecated Use `startAdornment` instead.
    */
   start?: React.ReactNode | string;
 
   /**
    * Content to display after the input.
+   *
+   * @deprecated Use `endAdornment` instead.
    */
   end?: React.ReactNode | string;
 
+  /**
+   * Element to display before the Button content.
+   */
+  startAdornment?: any;
```

Comment:
@amcdnl Can we keep that type `React.ReactNode | string`  for these properties?

Reply:
good catch - stupid copilot did that